unittests/test-contracts/get_table_test/get_table_test.cpp

```diff
@@ -15,6 +15,13 @@ void get_table_test::addnumobj(uint64_t input) {
       });
 }
 
+void get_table_test::modifynumobj(uint64_t id) {
+   numobjs numobjs_table( _self, _self.value );
+   numobjs_table.modify(numobjs_table.get(id), _self, [&](auto& message) {
```

Comment:
just a minor thing, but since we are using the name obj on the other places in this file, it would be better to change "message" to "obj"

Reply:
Done